### PR TITLE
General site cleanup -- part 1

### LINKED ISSUE #47: Remove geolocation button in search bar

### LINKED ISSUE #48: NV Items from Review of Production
Hi All - Per @jcahail's email to me, I have taken a look at the production site. So great to see it coming along. It's looking great! Attached are my general comments and notes. Many of these items are probably already on your list or being worked on as we speak. Let me know if you have any questions about this or want to chat. Thanks!

[ImpactView Feedback - 120517.docx](https://github.com/ImpactView/impact-view-philly/files/1531591/ImpactView.Feedback.-.120517.docx)

### LINKED ISSUE #49: Both hover elements should zindex on top of the map controls
See:
https://github.com/ImpactView/impact-view-philly/pull/45#pullrequestreview-81677623

### PR DESCRIPTION
**Pull request by CloudNiner**
## Overview

Addresses the remaining immediately addressable checkbox items on #48

## Demo (not necessarily comprehensive)

screen shot 2017-12-07 at 3 37 38 pm
screen shot 2017-12-07 at 3 30 18 pm
screen shot 2017-12-07 at 1 54 43 pm
screen shot 2017-12-07 at 1 54 29 pm
screen shot 2017-12-07 at 1 54 23 pm

## Notes

The remaining issues assigned to me, and open checkboxes on #48 have open questions that Jess is asking Neville about. I will address them separately in a followup PR.

## Testing

Ensure the app addresses each checked checkbox in #48 in addition to addressing the closed issues below.

Closes #47, Closes #48, Closes #49

### CODE REVIEW

File: app/scripts/museum/http-link-filter.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    'use strict';
+
+    /**
+     * Turn a string into an absolute url
+     *
+     * Basically, if it doesn't start with "http", prefix the http protocol and return new string
+     *
+     * This cheat allows us to safely account for site links that don't have the protocol prefix,
+     * but will also respect sites that do, e.g. if a site link is something like
+     * https://foo.com
```
Comment (KlaasH) [inline app/scripts/museum/http-link-filter.js:12]: Nice.

**Review by KlaasH — COMMENTED** [review]
Checkbox <span>#</span>2 (changing the language around the footer logos) doesn't seem to have been implemented.  Did that get spun into another issue?

Looks good otherwise.

**Comment by CloudNiner** [comment]
Updated to include changes to footer:

screen shot 2017-12-08 at 11 06 44 am

**Review by KlaasH — APPROVED** [review]
Issue #48 calls for the first bit of footer text to be "A Project Of" but this just says "Project Of".

Other than that, LGTM.  :+1:

**Comment by CloudNiner** [comment]
👍  updated.

**Pull request merged**